classes/Cart.php

```diff
@@ -1383,8 +1383,8 @@ public function getProductQuantity($idProduct, $idProductAttribute = 0, $idCusto
             $secondUnionSql .= $customizationJoin;
         }
         $commonWhere = '
-            WHERE cp.`id_product_attribute` = ' . (int) $idProductAttribute . '
-            AND cp.`id_customization` = ' . (int) $idCustomization . '
+            WHERE cp.`id_product_attribute` = ' . (int) $idProductAttribute .
+            ($idCustomization !== null ? ' AND cp.`id_customization` = ' . (int) $idCustomization : '') . '
```

Comment:
I wonder if you should do this?
```suggestion
            (!empty($idCustomization) ? ' AND cp.`id_customization` = ' . (int) $idCustomization : '') . '
```
This would remove the condition when `$idCustomization === 0` as well, but I'm not sure, do we want this extra condition when the value is zero? And only remove it specifically for null value?

Reply:
Yep, we want to check if `$idCustomization` is null and not an integer.
As I explained in my previous answer, 0 and null scenarios are not the same!